### PR TITLE
ci: fix setup kong scripts

### PR DESCRIPTION
**Pull request by pmalek**
Currently setup script are very fragile and susceptible to the state of the environment.

For instance when docker network used for tests already exists then user is greeted with 

```
Error response from daemon: network with name kong-test already exists
```

When that is sorted `./.ci/setup_kong_ee.sh` prints out:

```
d49438e56decc80a8ec362b515731cd551ee1536571eef7fb55e59f6f3ade158
./.ci/setup_kong_ee.sh: line 37: conditional binary operator expected
```

This PR aims to fix that and to extract some small utility functions to `./ci/_common.sh` which is then sourced in `setup_kong_ee.sh` and `setup_kong.sh`

### CODE REVIEW

File: .ci/_common.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+function create_network()
+{
+  # create docker network if it doesn't exist
+  if [[ -z $(docker network ls --filter name=${NETWORK_NAME} -q) ]]; then
+    docker network create ${NETWORK_NAME}
+  fi
+}
+
+function waitKongAPI() {
```
Comment (czeslavo) [inline .ci/_common.sh:25]:
```suggestion
# Usage: waitKongAPI <waitPeriodSeconds>
function waitKongAPI() {
```

**Review by czeslavo — APPROVED** [review]
🧹 Nice and tidy!

File: .ci/_common.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+function create_network()
+{
+  # create docker network if it doesn't exist
+  if [[ -z $(docker network ls --filter name=${NETWORK_NAME} -q) ]]; then
+    docker network create ${NETWORK_NAME}
+  fi
+}
+
+# Usage: waitKongAPI <waitPeriodSeconds>
+function waitKongAPI() {
```
Comment (pmalek) [inline .ci/_common.sh:26]:
```suggestion
function waitKongAPI()
{
```

**Comment by pmalek** [comment]
@czeslavo PTAL 🙏

**Pull request merged**